Doc/source/capabilities.rst

```diff
@@ -59,8 +59,9 @@ many models compatible with the Space Weather Modeling Framework,
 including the BATS-R-US global MHD model and the RAM-SCB ring current
 model.
 
-:mod:`~spacepy.omni` provides ready access to the OMNI near-Earth
-solar wind dataset, useful for model inputs.
+:mod:`~spacepy.omni` provides ready access to the `OMNI
+<https://omniweb.gsfc.nasa.gov/>`_ near-Earth solar wind dataset,
```

Comment:
Do we want to mention the Qin-Denton parameters (used for magnetic field models) here?

Reply:
I think it can go either way. The omni docs do mention it up-front so I think this is enough to get people in the right direction, but if you can think of pithy language ("including in processed forms" or something) I'd be game.

Reply:
I want to write something, but honestly let's just merge. I think this is sufficiently discoverable if someone is actually looking for it..